buildtools/build_wxwidgets.py

```diff
@@ -511,10 +523,18 @@ def main(wxDir, args):
     if options.extra_make:
         args.append(options.extra_make)
 
-    if not sys.platform.startswith("win"):
+    if sys.platform.startswith("win"):
+        if nmakeCommand.startswith('cmake'):
```

Comment:
can do:

```python
if sys.platform.startswith("win") and nmakeCommand.startswith('cmake'):
```

Reply:
No I cannot.

```python

    if sys.platform.startswith("win"):
        if nmakeCommand.startswith('cmake'):
            args.insert(0, '-GNinja')
    else:
        args.append("--jobs=" + options.jobs)
```

because if cmake or ninja is not available then cmake.exe doesn't get set to the nmake command so if the platform is windows we do not want to ad a --jobs to the nmake.exe command bcause it doesn't support this switch.